tests/rpmbuild.at

```diff
@@ -2451,6 +2453,105 @@ has_bcond(normally_on)
 [])
 RPMTEST_CLEANUP
 
+
+
+AT_SETUP([bcond_override_default macros])
+AT_KEYWORDS([bcond build])
+RPMDB_INIT
+
+# check bcond_override_default by defining
+AT_CHECK([
```

Comment:
AT_CHECK is deprecated in favor of RPMTEST_CHECK. It's currently just an alias so it works the same but for consistency, we should use the latter.